### ISSUE TITLE
0.2R Bug Report

### ISSUE THREAD

**Issue by 6h49af** [phdlee/uvk5cec#23]
1.We lost VOX by long press keypad-7 or F+7 in 0.2R Version.

2.CW mode tx always busy, in menu BusyCL : LOW, cannot be changed.

**Comment by 6h49af** [comment]
![3ECA980A-D0FC-431D-B072-D0EF7722DF8D](https://github.com/phdlee/uvk5cec/assets/152593919/bd01bd47-ed08-4a1d-bacb-dbb70386967c)
![D700E426-7636-4F2A-A04F-86766428B2DE](https://github.com/phdlee/uvk5cec/assets/152593919/a6973abe-03ae-4238-87b6-1d3f591864a4)

**Comment by phdlee** [comment]
I am very surprised by your feedback.
You found it a day after release. Please continue testing when next version
release.
you're right. The VOX function shortcut key function has been removed. (For
the Spectrum in next version) The activation/deactivation of the VOX
function can be set through the menu. The VOX function is maintained.

Many people talked about the need for Spectrum, so I decided to create
Spectrum with a small program size.
Existing programs take up about 5~6KByte of program space. I plan to aim
for a spectrum with a size of 1-2KByte.

If this is successful, FM Radio will also be changed to a smaller radio.
(Currently 4.3Kbyte, target is 1~2KByte)

And the 'BusyCL' function was disabled in version 0.2B. What appears in the
menu is incorrect. I will remove it from the menu in the next version.

Thanks for Excellent Testing
Ian KD8CEC

**Comment by 6h49af** [comment]
Although the side keys can also activate VOX, but key7 has vox symbol on it, which is more suitable for this function. Maybe you can activate Spectrum function by side keys long press setting.

**Comment by DL8LDN** [comment]
By the way: is it possible to create an adjustable VOX Delay down to 0,5 sec.? Will be good for digital modes coming in via microphone input. If you can do this you are my hero, hi 73 de DL8LDN

**Comment by 6h49af** [comment]
> By the way: is it possible to create an adjustable VOX Delay down to 0,5 sec.? Will be good for digital modes coming in via microphone input. If you can do this you are my hero, hi 73 de DL8LDN

I’m using VOX with a audio line, all OK.

**Comment by DL8LDN** [comment]
> > By the way: is it possible to create an adjustable VOX Delay down to 0,5 sec.? Will be good for digital modes coming in via microphone input. If you can do this you are my hero, hi 73 de DL8LDN
> 
> I’m using VOX with a audio line, all OK.

Yes it works but the time after there is no audio anymore and the trx switch 
to rx is too long for some modes like packet or VARA FM. VOX delay now is
good for phone.
